src/main/java/de/rwth/idsg/steve/config/SecurityConfiguration.java

```diff
@@ -61,11 +60,31 @@
  * @since 07.01.2015
  */
 @Slf4j
+@RequiredArgsConstructor
 @Configuration
 @EnableWebSecurity
 @Conditional(SteveProdCondition.class)
 public class SecurityConfiguration {
 
+    private final UserDetailsManager userDetailsManager;
+
+    @PostConstruct
+    public void postConstruct() {
+        String userName = SteveConfiguration.CONFIG.getAuth().getUserName();
+        if (userDetailsManager.userExists(userName)) {
+            return;
+        }
+
+        var user = User
+            .withUsername(userName)
+            .password(SteveConfiguration.CONFIG.getAuth().getEncodedPassword())
```

Comment:
Why not make it optional and generate a default password which will be logged like MySQL is doing.

Reply:
why? we are migrating from a state with a known username and pwd from properties file into DB. i can recreate the same world in DB. it would then not break installations. the data will be moved to database.

Reply:
Same idea as the username: "convention over configuration".
But it is not relevant here.